ICSharpCode.Decompiler.Tests/TestCases/Pretty/QualifierTests.cs

```diff
@@ -308,9 +308,9 @@ private void ParameterConflictsWithTypeName(string[] Array)
 
 		private void LocalConflictsWithTypeName()
 		{
-			for (int i = 0; i < 10; i++)
+			for (int j = 0; j < 10; j++)
 			{
-				QualifierTests.i.Test();
+				i.Test();
```

Comment:
hmm... doesn't this change defeat the purpose of the test?

Reply:
I would like to keep the test, but it seems we cannot actually test the case where a local variable conflicts with a type name, because `AssignVariableName` will no longer use nested type names as variable names :)